src/pages/community/release_procedure.md

```diff
@@ -61,9 +61,10 @@ Simplified release procedure:
 2. Stabilize the release by fixing test failures and bugs only
 3. Tag update release for a new version to prepare a release candidate, e.g `v1.3.0-1` for RC1
 4. Update the CHANGELOG
-5. Configure [release-configs.json](https://github.com/apache/yunikorn-release/tree/master/tools/release-configs.json)
-6. Run script [build-release.py](https://github.com/apache/yunikorn-release/tree/master/tools/build-release.py) to generate source code tarball, checksum and signature.
-7. Voting and releasing the candidate
+5. Update the DOAP file
```

Comment:
The release date is not known until we have voted and approved the release.

Reply:
Added this section under `Website updates for a new release` as suggested.